charts/pega/KafkaClusterRequirement.md

```diff
@@ -19,30 +19,33 @@ Production  | 4 cores | 16Gi   | 200G*      | At least 3
 * Above configuration would support up to 1000 kafka partitions, increase resources accordingly if kafka partitions are more.
 * Define appropriate quotas on network bandwidth and request rate if you want to share your kafka cluster across different environments.
 
-##### Miscellaneous configuration
-* message.max.bytes=5000000
+#### Miscellaneous configuration
+* message.max.bytes=5000000 
+  This is the default maximum message size supported, if you want to increase this value then pass the following jvm arguments to pega tiers as well
+  -Dstream.producer.max.request.size=<Max-message-size> -Dstream.producer.buffer.memory=<Max-message-size>
+  See more details about JVM arguments [here](README.md#jvm-arguments)
 * unclean.leader.election.enable=false
 * auto.create.topics.enable=false
 
 For best practices, see [this page.](https://docs.pega.com/decision-management/87/best-practices-stream-service-configuration)
 
-#### Security
+### Security
 
 Pega supports SSL for encryption of traffic as well as authentication to communicate with external Kafka. 
 
 In order to secure, mount necessary certificates(trustStore and keyStore) during the Pega Platform deployment. For details, see [this section.](README.md#optional-support-for-providing-credentialscertificates-using-external-secrets-operator)
 
 You may also securely pass settings like trustStorePassword,keyStorePassword, and jaasConfig through a secret in an external secret operator. For details, see [this section.](README.md#optional-support-for-providing-credentialscertificates-using-external-secrets-operator)
 
-### Kafka Permissions
+#### Permissions
 If external kafka is configured with authentication and authorization through Kafka Access control lists then Pega requires following user permissions
```

Comment:
Let's clean up the context a little>>
When you use an externalized kafka configuration, Pega requires the following user permissions to support your authentication and authorization using kafka Access Control lists: